### PR TITLE
[SOFT-294] Create telemetry makefile + get Travis working

### PR DESCRIPTION
**Pull request by hewittmcg**

### CODE REVIEW

**Comment by hewittmcg** [comment]
should be good to go

File: fan_control/fan_control.py [hunk 1]
```diff
@@ -18,31 +19,38 @@
 fan.value = 0
 
 # Returns PWM duty cycle
+
+
 def pwm_curve(temp):
```
Comment (AveryChiu64) [inline fan_control/fan_control.py:22]: I'm not really a big fan of having two newlines between the comment and function. The comment just looks out of place like this. Is this how the formatting is supposed to be?

**Review by AveryChiu64 — CHANGES_REQUESTED** [review]
The code for the fan and the formatting looks good. Could you see why there are two new lines between comments above a function? I think its sort of strange.

File: fan_control/fan_control.py (hunk 1 above)
Comment (hewittmcg) [reply fan_control/fan_control.py:22]: Oh yeah, that does look weird.  I googled around quickly and it seems like the [standard](https://www.python.org/dev/peps/pep-0257/ ) is to have a docstring inside the function directly below the function like this:

`def pwm_curve(temp): `
`''' Returns PWM duty cycle '''`

I'll take a look when I get a sec later today and see if that works with the formatter/if there's a way around it.

File: fan_control/fan_control.py [hunk 2]
```diff
@@ -17,32 +18,35 @@
...
         return 0
     else:
         return (1 - (PI_MAX_TEMP_C - temp) / (PI_MAX_TEMP_C - FAN_ON_TEMP_C))
 
-# Check fan temp + change pwm if needed
+
 def fan_control():
-  # Get CPU temp
-  temp = subprocess.run(["vcgencmd", "measure_temp"], stdout = subprocess.PIPE)
+    ''' Get CPU temp + change fan PWM as needed '''
```
Comment (hewittmcg) [inline fan_control/fan_control.py:31]: @AveryChiu64 changed comments to this.  I actually think this isn't _entirely_ with convention, but not having a space between the ''' and the text looks weird to me.  Lmk if this convention works for you

Comment (AveryChiu64) [reply fan_control/fan_control.py:31]: This looks good

**Review by AveryChiu64 — APPROVED** [review]

**Pull request merged**